app/Libraries/Elasticsearch/Search.php

```diff
@@ -257,14 +259,21 @@ private function getDatadogTags()
 
     private function handleError(ElasticsearchException $e, string $operation)
     {
+        $err = json_decode($e->getMessage(), true);
+
+        if (is_array($err) && str_starts_with($err['error']['caused_by']['reason'] ?? '', 'Failed to parse search_after value for field ')) {
+            $e = new InvalidCursorException();
+        }
+
         $tags = $this->getDatadogTags();
         $tags['class'] = get_class($e);
 
-        // Only report non query timeout errors to Sentry.
         // Printing the entire exception to log makes the breadcrumb too large to be sent to Sentry (16kb limit)
         // so we're only printing the message.
```

Comment:
Looking again, I'm not sure I understand this comment ಠ_ಠ The Sentry logging function only gets raw `$e`...

Reply:
Pretty sure this was for debugging timeouts when they were more common and having all relevant errors in the log file instead of bouncing between the log and sentry to find if a different exception class got thrown.

Logging after capture just causes the breadcrumb to go into the next captured error in the request.

This can probably be removed now, though 🤔